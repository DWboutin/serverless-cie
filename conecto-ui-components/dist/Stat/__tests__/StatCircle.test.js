"use strict";

var _interopRequireDefault = require("@babel/runtime/helpers/interopRequireDefault");

var _react = _interopRequireDefault(require("react"));

var _enzyme = require("enzyme");

var _Stat = _interopRequireDefault(require("../../Stat"));

var _Icon = _interopRequireDefault(require("../../Icon"));

describe('StatCircle', () => {
  it('should match snapshot', () => {
    const wrapper = (0, _enzyme.shallow)(_react.default.createElement(_Stat.default, {
      icon: _react.default.createElement(_Icon.default, {
        type: "notification"
      }),
      title: "Taux d'accceptation",
      notice: "+ 6.5% / mois pr\xE9c\xE9dent",
      number: 66
    }));
    expect(wrapper).toMatchSnapshot();
  });
  it('should have the prefix class', () => {
    const wrapper = (0, _enzyme.shallow)(_react.default.createElement(_Stat.default, {
      icon: _react.default.createElement(_Icon.default, {
        type: "notification"
      }),
      title: "Taux d'accceptation",
      notice: "+ 6.5% / mois pr\xE9c\xE9dent",
      number: 66
    }));
    expect(wrapper.hasClass('to-stat-circle')).toEqual(true);
  });
});