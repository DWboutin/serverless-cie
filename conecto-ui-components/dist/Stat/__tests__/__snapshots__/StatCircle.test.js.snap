// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StatCircle should match snapshot 1`] = `
<CardStat
  className="to-stat-circle"
  icon={
    <Icon
      type="notification"
    />
  }
  title="Taux d'accceptation"
  type="bordered"
>
  <div
    className="progress-circle p0 false"
  >
    <span>
      0
      %
    </span>
    <div
      className="left-half-clipper"
    >
      <div
        className="first50-bar"
      />
      <div
        className="value-bar"
      />
    </div>
  </div>
  <div
    className="notice"
  >
    + 6.5% / mois précédent
  </div>
</CardStat>
`;
